moped-editor/src/views/projects/projectView/ProjectComponents/EditModeDialog.js

```diff
@@ -34,7 +39,9 @@ const EditModeDialog = ({ showDialog, editDispatch, onEditFeatures }) => {
   return (
     <Dialog open={showDialog} onClose={onClose}>
       <DialogTitle className={classes.dialogTitle}>
-        <h3>What do you want to edit?</h3>
+        <Typography className={classes.titleStyles}>
+          What do you want to edit?
+        </Typography>
```

Comment:
this is just to quiet a warning that says an h3 cant be a child of an h2 (Dialog Title is an h2)